src/ui/settings/ViewTipsAndTricks/TipsAndTricksSection.qml

```diff
@@ -0,0 +1,125 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+
+ColumnLayout {
+    id: tipsAndTricksSection
+
+    property var flickableParent
```

Comment:
nit: since we're only accessing the flickable's `width` property, maybe we just pass in the width instead of the whole component